pkg/query-service/utils/format.go

```diff
@@ -221,3 +222,16 @@ func getPointerValue(v interface{}) interface{} {
 		return v
 	}
 }
+
+func GetClickhouseColumnName(typeName string, dataType, field string) string {
+	if typeName == string(v3.AttributeKeyTypeTag) {
+		typeName = constants.Attributes
+	}
+
+	if typeName != string(v3.AttributeKeyTypeResource) {
+		typeName = typeName[:len(typeName)-1]
```

Comment:
This can result in panic when typeName is empty. I assume that doesn't happen as of now, but to that law, Anything that can go wrong will go wrong at some point later. You can address this separately.

Reply:
Sure